src/general/NewGameSettings.cs

```diff
@@ -284,6 +264,11 @@ public override void _Ready()
             backButton.Visible = false;
             checkOptionsMenuAdviceContainer.Visible = false;
         }
+
+        planetSettings.Connect(PlanetSettings.SignalName.LawkSettingsChanged,
+            new Callable(this, nameof(OnLawkPlanetSettingChanged)));
+        planetSettings.Connect(PlanetSettings.SignalName.LifeOriginSettingsChanged,
+            new Callable(this, nameof(OnLifeOriginPlanetSettingChanged)));
```

Comment:
We prefer making connections like this through the Godot editor so could you move these into the NewGameSettings.tscn file?